src/node_api.cc

```diff
@@ -741,8 +741,19 @@ napi_status napi_define_class(napi_env env,
     v8::Local<v8::String> property_name;
     CHECK_NEW_FROM_UTF8(isolate, property_name, p->utf8name);
 
+    // convert the properties from NAPI to v8 format
+    unsigned int attribute_flags = v8::None;
```

Comment:
at least without adding more casts etc to the or lines.  I had tried that first but.